README.md

```diff
@@ -1,29 +1,67 @@
-- a `CONTRIBUTING.md` outlining how to contribute to the project
-- an unmodified copy of `code-of-conduct.md` from this repo, which outlines community behavior and the consequences of breaking the code
-- a `LICENSE` which must be Apache 2.0 for code projects, or [Creative Commons 4.0] for documentation repositories, without any custom content
-- a `SECURITY_CONTACTS` with the contact points for the Product Security Team 
-  to reach out to for triaging and handling of incoming issues. They must agree to abide by the
-  [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
-  and will be removed and replaced if they violate that agreement.
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Security Special Interest Group
 
-## Community, discussion, contribution, and support
+Covers horizontal security initiatives for the Kubernetes project, including regular security audits, the vulnerability management process, cross-cutting security documentation, and security community management.
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+The [charter](charter.md) defines the scope and governance of the Security Special Interest Group.
 
-You can reach the maintainers of this project at:
+## Meetings
+*Joining the [mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-security) for the group will typically add invites for the following meetings to your calendar.*
+* Regular SIG Meeting: [Thursdays at 9:00 PT (Pacific Time)](https://zoom.us/j/9934z1184192?pwd=L25Tc0ZOL3FqU09KNERlTU12dFhTQT09) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1GgmmNYN88IZ2v2NBiO3gdU8Riomm0upge_XNVxEYXp0/edit?usp=sharing).
+  * [Meeting recordings](https://www.youtube.com/playlist?list=PL69nYSiGNLP1mXOLAc9ti0oX8s_ookQCi).
 
-- [Slack](http://slack.k8s.io/)
-- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-dev)
+## Leadership
 
-### Code of conduct
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
 
-Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
+* Ian Coldwater (**[@IanColdwater](https://github.com/IanColdwater)**), Twilio
+* Tabitha Sable (**[@tabbysable](https://github.com/tabbysable)**), Datadog
 
-[owners]: https://git.k8s.io/community/contributors/guide/owners.md
-[Creative Commons 4.0]: https://git.k8s.io/website/LICENSE
+## Contact
+- Slack: [#sig-security](https://kubernetes.slack.com/messages/sig-security)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-security)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fsecurity)
```

Comment:
Add an entry for sig-security issues - https://github.com/kubernetes/sig-security/issues